### DISCUSSION TITLE
Pi 3 Sound Glitches When Page Refreshed

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#648] [Fixed Issues] [answered]
With a Pi3 and 64 bit image, I get glitches in the sound when moving through the menus or refreshing a page. There is no problem with the Pi4.

rAudio 1 • 20211122
5.10.81-1 • aarch64
20211108-1
Raspberry Pi 3 Model B Rev 1.2
BCM2837 • 958MiB
4 @ 1.2GHz

**Comment by ceejayemm** [comment, marked as answer]
Sorry, can't confirm this on RPi 3A with20211122. No issue when moving between any screens or screen refresh.

rAudio 1 • 20211122
5.10.79-2 • aarch64
20211108-1
Raspberry Pi 3 Model A Plus Rev 1.0
BCM2837B0 • 442MiB
4 @ 1.4GHz

Chris

**Comment by R101G** [comment]
Thanks for your reply @ceejayemm . It looks as if it's a performance issue, and the original BCM2837 model B is struggling a bit.
